### PR TITLE
add module to load environment variables from aws secretsmanager

### PR DESCRIPTION
**Pull request by drsherluck**
Allows application to load secrets into their process's environment at using secrets manager.
Future improvement would be to use caching system that refreshes the secrets every hour (see https://docs.aws.amazon.com/secretsmanager/latest/userguide/retrieving-secrets_cache-python.html)

### CODE REVIEW

File: TracefyClients/secretsmanager.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+):
+    """
+    Loads key value pairs defined in the secret ids inside the comma separated
+    environment variable (default is AWS_SECRETSMANAGER_SECRET_IDS)
+
+    Call secretsmanager_loadenv at the start of your application
+    """
+
+    env = secretsmanager_values(secrets, region_name)
+    load_dictionary_as_env(env, overwrite_existing_keys)
```
Comment (drsherluck) [inline TracefyClients/secretsmanager.py:147]: We should probably verify that the keys are matching the standard environment key `^[A-Z_][A-Z0-9_]*` or do like Kubernetes and allow lowercase characters as well `[A-Za-z_][A-Za-z0-9_]*`.
*I think k8s also converts all lowercase characters to uppercase and replaces invalid characters into `_`*

File: requirements.txt [hunk 2]
```diff
@@ -3,11 +3,15 @@ Flask-Cors==5.0.0
 python-dotenv==1.0.1
 sentry-sdk==2.13.0
 marshmallow==3.22.0
-pytest==8.3.2
```
Comment (drsherluck) [inline requirements.txt:6]: This does not make sense to be added to the `requirements.txt` file as it should be only used for testing.

**Review by kftracefy — APPROVED** [review]
LGTM

**Review by henkytracefy — APPROVED** [review]
lgtm

**Pull request merged**